pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -598,7 +598,20 @@ type MonSpec struct {
 	// VolumeClaimTemplate is the PVC definition
 	// +kubebuilder:pruning:PreserveUnknownFields
 	// +optional
-	VolumeClaimTemplate *v1.PersistentVolumeClaim `json:"volumeClaimTemplate,omitempty"`
+	VolumeClaimTemplate *VolumeClaimTemplate `json:"volumeClaimTemplate,omitempty"`
```

Comment:
```suggestion
	VolumeClaimTemplate *VolumeClaimTemplate `json:"volumeClaimTemplate,omitempty"`
```

why we are not using default ones?

Reply:
Do you mean PVC? The reason is that this includes a `Status` field that we can't make use of in our CRD, and it generates into several CRDs. We can remove it to save CRD space.

Reply:
got it.